salt/utils/__init__.py

```diff
@@ -3403,3 +3255,89 @@ def get_colors(use=True, theme=None):
         'Oxygen. This warning will be removed in Salt Neon.'
     )
     return salt.utils.color.get_colors(use=use, theme=theme)
+
+
+def gen_state_tag(low):
+    '''
+    Generate the running dict tag string from the low data structure
+
+    .. deprecated:: Oxygen
+    '''
+    # Late import to avoid circular import.
+    import salt.utils.versions
+    import salt.utils.state
+    salt.utils.versions.warn_until(
+        'Neon',
+        'Use of \'salt.utils.gen_state_tag\' detected. This function has been '
+        'moved to \'salt.utils.state.gen_tag\' as of Salt Oxygen. This warning '
+        'will be removed in Salt Neon.'
+    )
+    return salt.utils.state.gen_tag(low)
+
+
+def search_onfail_requisites(sid, highstate):
+    '''
+    For a particular low chunk, search relevant onfail related states
+
+    .. deprecated:: Oxygen
+    '''
+    # Late import to avoid circular import.
+    import salt.utils.versions
+    import salt.utils.state
+    salt.utils.versions.warn_until(
+        'Neon',
+        'Use of \'salt.utils.search_onfail_requisites\' detected. This function'
```

Comment:
nit: In this and the next three calls to `warn_until`, add a trailing space like in `gen_state_tag` to prevent the words `function` and `has` from running together.